src/nncf/torch/model_graph_manager.py

```diff
@@ -371,6 +372,46 @@ def get_weight_channel_axes(metatype: type[OperatorMetatype], ndims: int, input_
     return (0,)
 
 
+def get_weight_compression_reduction_axes(
+    metatype: OperatorMetatype, weight_port_id: int, ndims: int
+) -> Optional[list[int]]:
+    """
+    Returns reduction axes for the given parameters without axes that corresponds to weight channels of a node with the
+    given metatype.
+
+    :param metatype: The metatype of the operator node containing the weight.
+    :param weight_port_id: The index of the input port corresponding to the weight tensor.
+    :param ndims: Number of dimensions in the weight tensor.
+    :return: list of axes to reduce over, or None if no reduction axes are determined.
+    """
+    reduction_axes = None
```

Comment:
After a consideration, I think it would be clearer to use `return` statement instead of a variable here (and no eiif statements)
and please update the return typehint